python/foglamp/services/common/microservice.py

```diff
@@ -67,8 +77,8 @@ def __init__(self, default_config):
             })
             self._core_microservice_management_client.create_configuration_category(config_payload)
             config = self._core_microservice_management_client.get_configuration_category(category_name=category)
-            host = config['management_host']['value']
-            # -----
+            is_local_services = True if config['local_services']['value'].lower() in ['true', 't', '1', 1] else False
+            host = '127.0.0.1' if is_local_services is True else '0.0.0.0'
```

Comment:
if is_local_services is True IS more concise and equivalent to `if is_local_services`